src/main/java/tuteez/logic/commands/AddCommand.java

```diff
@@ -75,9 +75,9 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_DUPLICATE_PERSON);
         }
 
-        Set<Lesson> lessonSet = toAdd.getLessons();
+        List<Lesson> lessonLst = toAdd.getLessons();
```

Comment:
Nice work with changing the sets to a list so it is easier for me to display on the right side!